src/index.ts

```diff
@@ -40,16 +45,24 @@ The following environment variables are used:
 
 function verifyAndGetEnvVars() {
   const manifestUri = process.env[MANIFEST_URI_KEY];
+  const s3Bucket = process.env[S3_BUCKET_KEY];
+  const s3Path = process.env[S3_PATH_KEY];
```

Comment:
i was thinking about this first, but we also [allow MANIFEST_URI to be a directory as well ](https://github.com/mongodb/docs-search-transport/blob/main/src/SearchIndex/util.ts#L554) and in this case, we would already need to provide a separate bucket name and path as well

Reply:
Oh, interesting. Thanks for pointing out that use case. I think it's awkward either way since if a local directory is being used as the URI, the new endpoint will return a list of S3 URLs that may not even be reflective of the manifests in the directory being used. For example, manifests in the directory can have manifests that don't exist in the S3 bucket, and vice-versa.

That being said, this isn't a big deal and was just a small concern. I'm okay with keeping as-is